### ISSUE TITLE
Several (at least 2) simultaneous translations

### ISSUE THREAD

**Issue by iG8R** [immersive-translate/immersive-translate#1440]
### 您希望的更新和改进是什么 | Update or Improve

Could you please implement the possibility to translate with the several services simultaneously, e.g.
```
original text
translated text using service#1
translated text using service#2
```
In the case of very specific and hard to translate or ambiguous texts, e.g. medicine, chemistry, physics, Chinese novels:) etc., it turns out that the one translation service does well with only one part of the text on the page, and another translation service does well with only another part.
At such times, I always have a thought - how wonderful it would be to be able to see translations from several translation services at once, instead of constantly switching between one, another, and sometimes even a third one.

**Comment by keyboard3** [comment]
Adding multiple translation services to the original web page will increase the complexity of the program. In the future, an input box translation page will be provided to enable you to compare the translation results of multiple translation services.

**Comment by iG8R** [comment]
Thanks a lot!
Could you please elaborate a little what will it be look like `an input box translation page will be provided to enable you to compare the translation results of multiple translation services`?

**Comment by keyboard3** [comment]
[This page](https://app.immersivetranslate.com/text/) supports comparison of multiple translation services。

**Comment by iG8R** [comment]
> [This page](https://app.immersivetranslate.com/text/) supports comparison of multiple translation services。

No, that's absolutely not what the request was about.
On [this page](https://app.immersivetranslate.com/text/) the user needs to enter every questionable sentence separately, but the request was about the whole page, because, for example, novels have many such sentences with ambiguous wording during translation with the only 1 translation service.
Please, re-open this issue.

**Comment by theowenyoung** [comment]
It's not very costly to support this technically, but I'm not sure how I'm going to adapt this logic in the UI. This seems like a UI logic nightmare, do you have any ideas?

**Comment by iG8R** [comment]
In my mind I see the following:
In the Settings page add additional sub-page, e.g. "Multiservice Translation"
On this page:
- list all available services with the checkboxes next to each of them to allow the user to pick up needed services
- place the "Translation display style", "Customize colors and sizes", "Set font", "Preview all themes" options for each of the corresponding services
- place input fields for each service to determine the sequence of appearance of the translations

![image](https://github.com/immersive-translate/immersive-translate/assets/11407417/c991159a-912c-4a38-9a27-d15b1c871aca)

**Comment by theowenyoung** [comment]
Yes, this UI interaction is a good idea for the feature you want to implement.

But on the other hand I can imagine the cost of implementing this UI itself, and the software complexity that comes with implementing this feature, so let's  see if more people will expect this feature.

**Comment by iG8R** [comment]
Still waiting with great hope that this feature will be implemented in the near future.
